src/Shaders/pbr.fragment.fx

```diff
@@ -659,4 +755,12 @@ void main(void) {
 	//// Emissive Color
 	//vec2 test = vEmissiveUV * 0.5 + 0.5;
 	//gl_FragColor = vec4(test.x, test.y, 1.0, 1.0);
+

```

Comment:
There are debug switch :-) just uncomment and you see the channel, I ll remove once Spector supports it.